### PR TITLE
Clarify where annotation should be inserted

### PR DESCRIPTION
**Pull request by kramvan1**
Explain the default merge scrape path:port.
Add some detail on where the annotations should go.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by kramvan1** [comment]
@howardjohn   I think this will help here, thoughts?

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To simplify configuration, Istio has the ability to control scraping entirely by
 While `prometheus.io` annotations are not a core part of Prometheus, they have become the de facto standard to configure scraping.
 {{< /tip >}}
 
-This option is enabled by default but can be disabled by passing `--set meshConfig.enablePrometheusMerge=false` during [installation](/docs/setup/install/istioctl/). When enabled, appropriate `prometheus.io` annotations will be added to all workloads to set up scraping. If these annotations already exist, they will be overwritten. With this option, the Envoy sidecar will merge Istio's metrics with the application metrics.
+This option is enabled by default but can be disabled by passing `--set meshConfig.enablePrometheusMerge=false` during [installation](/docs/setup/install/istioctl/). When enabled, appropriate `prometheus.io` annotations will be added to all data plane pods to set up scraping. If these annotations already exist, they will be overwritten. With this option, the Envoy sidecar will merge Istio's metrics with the application metrics. The merged metrics will be scraped by default from `/stats/prometheus:15020`.
```
Comment (howardjohn) [inline content/en/docs/ops/integrations/prometheus/index.md:33]: Its not really "by default from `/stats/prometheus:15020`" right? Its always scraped from this point, which will then forward to envoy/app

File: content/en/docs/ops/integrations/prometheus/index.md [hunk 2]
```diff
@@ -48,11 +48,17 @@ The built-in demo installation of Prometheus contains all the required scraping
...
-* `prometheus.io/path` determines the path to scrape metrics at. Defaults to `/metrics`.
-* `prometheus.io/port` determines the port to scrape metrics at. Defaults to `80`.
+This configuration will add scrape job configurations for the control plane, as well as for all Envoy sidecars. Additionally, a job is configured to scrape application metrics for all data plane pods with relevant `prometheus.io` annotations:
+
+{{< text yaml >}}
+spec:
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: true   # determines if a pod should be scraped. Default to true.
```
Comment (howardjohn) [inline content/en/docs/ops/integrations/prometheus/index.md:58]: I don't think it defaults to true?

File: content/en/docs/ops/integrations/prometheus/index.md (hunk 1 above)
Comment (kramvan1) [reply content/en/docs/ops/integrations/prometheus/index.md:33]: Sure, will remove the default wording.

File: content/en/docs/ops/integrations/prometheus/index.md (hunk 2 above)
Comment (kramvan1) [reply content/en/docs/ops/integrations/prometheus/index.md:58]: Sure will reword that.

**Comment by frankbu** [comment]
@googlebot

**Pull request merged**